src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -149,6 +149,12 @@ Blank()
 >> DateDiff(DateTimeValue(ParseJSON("""2011-01-15T08:00:00.000Z""")), DateTimeValue(ParseJSON("""2011-01-30T08:00:00.000Z""")))
 15
 
+>> DateDiff(DateTimeValue(ParseJSON("""2011-01-15T08:00:00Z""")), DateTimeValue(ParseJSON("""2011-01-30T08:00:00.000""")))
```

Comment:
>DateDiff(DateTimeValue(ParseJSON("""2011-01-15T08:00:00Z""")), DateTimeValue(ParseJSON("""2011-01-30T08:00:00.000"""))) [](http://example.com/codeflow?start=3&length=119)

This will not be true in all time zones. For example, in Hawaii (UTC-10), the first value will be 2011-01-14 22:00:00 and the second one will be 2011-01-30 08:00:00, which will give a result of 16

Reply:
Removed Z time zone tests, which will fail in certain time zones